internal/service/resourcepolicy/model_test.go

```diff
@@ -151,3 +150,24 @@ func Test_NewUserMetadataObjectTypeWithNilArg(t *testing.T) {
 	unit.AssertDiagsOK(t, diags)
 	assert.Equal(t, types.ObjectNull(resourcepolicy.UserMetadataObjectType.AttrTypes), obj)
 }
+
+func Test_NewTFPoliciesModelToSDK(t *testing.T) {
+	ctx := context.Background()
+	policies := []resourcepolicy.TFPolicyModel{
```

Comment:
knit: we're using an intermediate name here, not ResourcePolicy or remove it completely, I would favor TFModel, alternatively TFResourcePolicyModel would be "more correct" if you want to keep the resource name, same for other structs

Reply:
Changed in c1b1adb5d07aad9d03b330d76b42d7ef7a5c77a0 WDYT?